state/futures/hooks.ts

```diff
@@ -47,11 +83,23 @@ export const usePollDashboardFuturesData = () => {
 	const markets = useAppSelector(selectMarkets);
 	const wallet = useAppSelector(selectWallet);
 	const crossMarginAddress = useAppSelector(selectCrossMarginAccount);
+	const networkSupportsCrossMargin = useAppSelector(selectCrossMarginSupportedNetwork);
+
+	useFetchAction(fetchCrossMarginAccount, {
+		dependencies: [networkId, wallet],
+		disabled: !wallet || !networkSupportsCrossMargin,
+	});
 
 	usePollAction('fetchSharedFuturesData', fetchSharedFuturesData, {
 		dependencies: [networkId],
 		intervalTime: 60000,
 	});
+	usePollAction('fetchFundingRates', fetchFundingRates, {
```

Comment:
We don't need this anymore since the value reported by the V2 contracts is already a smoothed average rate for 24h. We can simply divide the rate by 24 and report that rate and avoid this complicated calculation.